frontend/src/components/Uploader/ProvidersUplodader.tsx

```diff
@@ -62,15 +64,22 @@ const COLUMNS = [
 ]
 
 let parse = parseRows(COLUMNS, ([address, city, state, zip,
-  npi, language1, language2, language3, specialty]) => {
-
-  let fullAddress = `${address}, ${city}, ${state} ${normalizeZip(zip!)}`
+  latitude, longitude, npi, language1, language2, language3, specialty]) => {
+  let finalAddress = `${address}, ${city}, ${state} ${normalizeZip(zip!)}`
   let languages = [language1, language2, language3].filter(Boolean) as string[]
-
   return {
-    address: fullAddress,
+    address: finalAddress,
+    lat: parseF(latitude),
+    lng: parseF(longitude),
```

Comment:
`longitude || undefined`

Reply:
I tried but parseFloat refuses latitude and longitude as they could be null... @bcherny

Reply:
Ah good call! Thanks typescript!

In that case I'd consider renaming your function to something more understandable, eg. `maybeParseFloat` and move it to utils/number.ts.